src/resources/icons/index.js

```diff
@@ -0,0 +1,8 @@
+const Icons = {
+  languageEn: require('./language-en.svg'),
+  languagePt: require('./language-pt.svg'),
+  letter: require('./letter-ic.svg'),
+  gatsby: require('./gatsby-icon.png'),
```

Comment:
Why not importing all these icons statically? You could import them at the top using ES Modules. By the way, if you need dynamic import, you can also use dynamic ES `import`, such as:

```suggestion
  gatsby: import('./gatsby-icon.png'),
```

Reply:
**Why not importing all these icons statically?**
→ Because I think it generates extra boilerplate. All I'm doing here is grouping all my icon references into a single Icons object. 
**If you need dynamic import, you can also use dynamic ES import**
→ Okay, I'll move to that syntax.